### PR TITLE
Feature/overview style

### PR DESCRIPTION
**Pull request by brycemara**
## Description
- Style Overview
- Pass specific id to room

## Related Issue
Issue #4 

## How Has This Been Tested?
- THIS IS NOT RESPONSIVE. 
- Looks great on the iPad but on my phone doesn't work. 
- No tests written yet.

## Screenshots (if appropriate):

Screen Shot 2021-02-26 at 2 15 32 PM

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

**Review by Estaffieri — APPROVED** [review]
Looks great! Thanks for that note in Rooms as well!

File: src/Overview/Overview.js [hunk 1]
```diff
@@ -1,38 +1,121 @@
...
-      </Button>
+        <View style={styles.backyard}>
+          <Text style={styles.text}>{backyard.name}</Text>
+        </View>
+      </TouchableOpacity>
     </View>
   );
 };
 
 const styles = StyleSheet.create({
```
Comment (cameronRomo) [inline src/Overview/Overview.js:58]: Very nice use of flex here!

File: src/Overview/Overview.js [hunk 2]
```diff
@@ -1,38 +1,121 @@
...
+    navigation.navigate("Room", { id: id });
   };
 
   return (
     <View style={styles.container}>
-      <Text>OVERVIEW</Text>
-      <Button
+      <TouchableOpacity
+        onPress={() => {
+          routeToRoom(kitchen.id);
```
Comment (cameronRomo) [inline src/Overview/Overview.js:18]: Nice! So from here, it seems like we can bring the useEffect hook into the various room components to resolve the API call upon landing in the component.

File: src/Overview/Overview.js [hunk 3]
```diff
@@ -1,38 +1,121 @@
...
-  //     </View>
-  //   );
-  // });
-  const buttonPress = () => {
-    props.navigation.navigate("Room");
+export default Overview = ({ navigation, route }) => {
+  let kitchen = route.params.rooms[0];
+  let livingRoom = route.params.rooms[1];
+  let backyard = route.params.rooms[2];
+  let bedroom = route.params.rooms[3];
```
Comment (Codo-Baggins) [inline src/Overview/Overview.js:8]: This really does look great! Going to have to explain these lines to me though, not sure whats happening here.

File: src/Overview/Overview.js [hunk 4]
```diff
@@ -1,38 +1,121 @@
...
+  backyard: {
+    borderRadius: 10,
+    borderWidth: 5,
+    borderStyle: "solid",
+    borderColor: "white",
+    backgroundColor: "#768243",
+    height: 300,
+    width: 700,
   },
 });
```
Comment (Codo-Baggins) [inline src/Overview/Overview.js:121]: Nice job with these styles!

**Review by Codo-Baggins — APPROVED** [review]
Great job!

**Pull request merged**